src/style.css

```
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
```

Remove redundant default styles
```suggestion
  font-weight: 300;
  src: url("./fonts/Avenir-Book.ttf") format("truetype");
}

@font-face {
  font-family: "Avenir";
  font-weight: 600;
  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
```